packs/journals/gm-screen.json

```diff
@@ -1812,6 +1812,28 @@
                 "controls": true,
                 "volume": 0.5
             }
+        },
+        {
+            "_id": "8gcp880pEWZ9VPnF",
+            "image": {},
+            "name": "Summon Trait",
+            "sort": 770735,
+            "src": null,
+            "system": {},
+            "text": {
+                "content": "<p>Spells with the summon trait can conjure creatures, typically ones with a particular trait. Such creatures can be found in Monster Core and similar books. Unless noted otherwise, the creature must be common, it gains the summoned trait (page 301), and it must appear in an unoccupied space in range large enough to contain it. The highest level of creature the spell can summon depends on the rank of the spell, as listed below. The spell can still summon a creature of a lower level if you so choose. These rules apply only to spells that have the summon trait; other spells that call or conjure items or beings but that don’t have the trait, like summon instrument, work as explained in the spell.</p><table class=\"pf2e\"><tbody><tr><th scope=\"col\" data-colwidth=\"281\"><p>Spell Rank</p></th><th scope=\"col\"><p>Maximum Creature Level</p></th></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">1st</p></td><td><p style=\"text-align:center\">-1</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">2nd</p></td><td><p style=\"text-align:center\">1</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">3rd</p></td><td><p style=\"text-align:center\">2</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">4th</p></td><td><p style=\"text-align:center\">3</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">5th</p></td><td><p style=\"text-align:center\">5</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">6th</p></td><td><p style=\"text-align:center\">7</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">7th</p></td><td><p style=\"text-align:center\">9</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">8th</p></td><td><p style=\"text-align:center\">11</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">9th</p></td><td><p style=\"text-align:center\">13</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">10th</p></td><td><p style=\"text-align:center\">15</p></td></tr></tbody></table>",
```

Comment:
```suggestion
                "content": "<p>Spells with the summon trait can conjure creatures, typically ones with a particular trait. Such creatures can be found in Monster Core and similar books. Unless noted otherwise, the creature must be common, it gains the summoned trait, and it must appear in an unoccupied space in range large enough to contain it. The highest level of creature the spell can summon depends on the rank of the spell, as listed below. The spell can still summon a creature of a lower level if you so choose. These rules apply only to spells that have the summon trait; other spells that call or conjure items or beings but that don’t have the trait, like summon instrument, work as explained in the spell.</p><table class=\"pf2e\"><tbody><tr><th scope=\"col\" data-colwidth=\"281\"><p>Spell Rank</p></th><th scope=\"col\"><p>Maximum Creature Level</p></th></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">1st</p></td><td><p style=\"text-align:center\">-1</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">2nd</p></td><td><p style=\"text-align:center\">1</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">3rd</p></td><td><p style=\"text-align:center\">2</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">4th</p></td><td><p style=\"text-align:center\">3</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">5th</p></td><td><p style=\"text-align:center\">5</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">6th</p></td><td><p style=\"text-align:center\">7</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">7th</p></td><td><p style=\"text-align:center\">9</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">8th</p></td><td><p style=\"text-align:center\">11</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">9th</p></td><td><p style=\"text-align:center\">13</p></td></tr><tr><td data-colwidth=\"281\"><p style=\"text-align:center\">10th</p></td><td><p style=\"text-align:center\">15</p></td></tr></tbody></table>",
```